.github/workflows/main.yaml

```diff
@@ -21,7 +18,7 @@ jobs:
           reporter: github-pr-check
           fail-on-error: true
 
-  unit-tests:
+  lint-unit:
```

Comment:
can you use the [reusable workflow here](https://github.com/charmed-kubernetes/workflows/blob/main/.github/workflows/lint-unit.yaml)

Reply:
yeah, that's just in a different PR
* https://github.com/charmed-kubernetes/charm-containerd/pull/66